dask_cudf/io/tests/test_csv.py

```diff
@@ -11,11 +11,18 @@ def test_read_csv(tmp_path):
     df = dask.datasets.timeseries(dtypes={"x": int, "y": int}, freq="120s").reset_index(
         drop=True
     )
-    df.to_csv(tmp_path / "data-*.csv", index=False)
+    # tmp_path is a PosixPath -- need to convert to string
+    stmp_path = str(tmp_path / "data-*.csv")
```

Comment:
I would have thought that dask.dataframe.to_csv could handle this.  Was this failing before?  If so, can I ask you to raise an issue on Dask?

Reply:
The issue here is not with dask.dataframe but rather the conditional check in `dask_cudf.read_csv`.  PosixPath is not iterable and thus:  `"://" in path` fails

Reply:
we could str() in read_csv ?

Reply:
In that case I recommend that we convert during that check

```python
if "://" in str(path):
    ...
```

That way users can still use Path objects

Reply:
Right, that